projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -483,4 +485,18 @@ private static function file_system_write_access() {
 
 		return \Automattic\Jetpack\Sync\Functions::file_system_write_access();
 	}
+
+	/**
+	 * Update the health check paths for a scheduled update.
+	 *
+	 * @param string           $id      The ID of the schedule.
+	 * @param object           $event   The event object.
+	 * @param \WP_REST_Request $request The request object.
+	 * @return bool
+	 */
+	public static function updates_health_paths( $id, $event, $request ) {
+		$schedule = $request['schedule'];
+		$paths    = $schedule['health_check_paths'] ?? array();
+		return Scheduled_Updates_Health_Paths::update( $id, $paths );
+	}
```

Comment:
Let's move this into the `Scheduled_Updates_Health_Paths` class to keep that functionality together

Reply:
Sure, moved!